src/main/java/com/to/ordermanagementservice/utils/TotalPriceCalculator.java

```diff
@@ -0,0 +1,24 @@
+package com.to.ordermanagementservice.utils;
+
+import com.to.ordermanagementservice.dto.OrderItemDetails;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public final class TotalPriceCalculator {
+    private TotalPriceCalculator() {
+    }
+
+    public static BigDecimal getTotalPriceForOrder(List<OrderItemDetails> orderItems) {
+        BigDecimal result = BigDecimal.ZERO;
+        if (orderItems == null) {
+            return result;
+        }
```

Comment:
you may add ` || orderItems.isEmpty()` to avoid redundant processing if a List is empty

Reply:
checking of empty list is add